ipn/ipnlocal/local.go

```diff
@@ -324,6 +330,10 @@ type LocalBackend struct {
 
 	// outgoingFiles keeps track of Taildrop outgoing files keyed to their OutgoingFile.ID
 	outgoingFiles map[string]*ipn.OutgoingFile
+
+	// lastSuggestedExitNode stores the last suggested exit node ID and name.
+	// lastSuggestedExitNode updates whenever the suggestion changes.
+	lastSuggestedExitNode lastSuggestedExitNode
```

Comment:
Do we have to reset this value when switching between user profiles? (different tailnets won't have the same exit nodes)

Reply:
decided on clearing the last suggested exit node everytime there's a profile switch